src/serac/numerics/equation_solver.cpp

```diff
@@ -49,6 +52,27 @@ class NewtonSolver : public mfem::NewtonSolver {
     return normEval;
   }
 
+  /// assemble the jacobian
+  void assembleJacobian(const mfem::Vector& x) const
+  {
+    CALI_CXX_MARK_FUNCTION;
+    grad = &oper->GetGradient(x);
```

Comment:
While we're at it, could we instrument this GetGradient call with some caliper annotations as well? This is such an expensive operation, it would be beneficial to have more information about where that time is being spent.